src/main/java/mate/academy/springboot/datajpa/dto/CategoryResponseDto.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.springboot.datajpa.dto;
+
+import lombok.Data;
+
+@Data
+public class CategoryResponseDto {
+    private Long id;
```

Comment:
Do you need send id in response?

Reply:
Yep, I want to return the ID of the saved element